### PR TITLE
get dask labextension to start k8s cluster, not local

### LINKED ISSUE #148: dask labextension not working in ocean
I am seeing this in my notebook pod logs:

```
404 GET /user/0000-0001-5999-4917/dask/clusters?1551920957641 (0000-0001-5999-4917@10.128.0.18) 2.41ms
```

after I click on the button to create a new cluster.

### PR DESCRIPTION
**Pull request by raphaeldussin**
fixes #148

### CODE REVIEW

**Comment by rabernat** [comment]
So the NASA image uses this docker image
https://github.com/pangeo-data/pangeo-cloud-federation/blob/1bbab64ec0c4dd14eb18c74fd7610e0f6f7ecd68/deployments/nasa/image/binder/Dockerfile#L1-L2

While ocean uses 
https://github.com/pangeo-data/pangeo-cloud-federation/blob/1bbab64ec0c4dd14eb18c74fd7610e0f6f7ecd68/deployments/ocean/image/binder/Dockerfile#L1-L2

What is the difference between the two? Does pointing to "onbuild" make things faster or better in some way?

**Comment by jhamman** [comment]
Using the onbuild image allows you to extend the image with repo2docker syntax. If you are going to use a postBuild, start, or extended environment.yaml, you'll need to use the onbuild image.

**Comment by raphaeldussin** [comment]
@jhamman @rabernat so we should start with nasa config and tune it from there. right?

File: deployments/ocean/image/binder/environment.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - ipyleaflet=0.10
+  - datashader=0.6
+  - requests=2.21
+  - pandas=0.24
+  - geopandas=0.4
+  - rasterio=1.0
+  - pip:
+    - rio-cogeo
+    - sat-search==0.2.0
+    - dask-labextension==0.3.1
```
Comment (jhamman) [inline deployments/ocean/image/binder/environment.yml]: So this is fine but its really overkill -- I don't really like how the nasa image is working at the moment. The idea is just to add packages not included in the base image. That may not be anything in your case. All this version pinning  kind of defeats the purpose of using the onbuild image.

File: deployments/ocean/image/binder/environment.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - boto3=1.9
+  - awscli=1.16
+  - hvplot=0.4
+  - geoviews-core=1.6
+  - ipyleaflet=0.10
+  - datashader=0.6
+  - requests=2.21
+  - pandas=0.24
+  - geopandas=0.4
+  - rasterio=1.0
```
Comment (rabernat) [inline deployments/ocean/image/binder/environment.yml]: I don't think any of this environment.yml should be necessary. I want all of this to live in pangeo-stacks.

File: deployments/ocean/image/binder/postBuild [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+jupyter serverextension enable --py nbserverproxy --sys-prefix
+jupyter labextension install @jupyter-widgets/jupyterlab-manager \
+                             @jupyterlab/hub-extension \
+                             @pyviz/jupyterlab_pyviz \
+                             jupyter-leaflet \
+                             dask-labextension
```
Comment (rabernat) [inline deployments/ocean/image/binder/postBuild]: All of the above should be done in pangeo-notebook in docker-stacks

**Review by rabernat — CHANGES_REQUESTED** [review]

**Comment by rabernat** [comment]
Does `pangeo-notebook-onbuild:2019.04.19 ` have all the same stuff that `pangeo-notebook:2019.04.19 ` does?

**Comment by rabernat** [comment]
So @raphaeldussin, what happens if you just drop environment.yml and the jupyter stuff in PostBuild?

**Comment by jhamman** [comment]
> Does pangeo-notebook-onbuild:2019.04.19 have all the same stuff that pangeo-notebook:2019.04.19 does?

Yes.

**Comment by raphaeldussin** [comment]
looks good? merge?

**Comment by jhamman** [comment]
LGTM!

**Pull request merged**